README.md

```diff
@@ -90,82 +94,19 @@ attempt to use newer ones, the build will fail. Presently, Rust doesn't allow
 to generate custom error messages so there doesn't appear to be a way to make
 such errors more friendly.
 
-Examples are providing in the [gtk-rs/examples](https://github.com/gtk-rs/examples) repository, you can find some tests showing off the functionality, these can be built and run as follows:
```

Comment:
No wait, the screenshots aren't local. So the `resources` directory is an even older artifact.